tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -526,6 +526,15 @@ public:
    RNTuple() = default;
    ~RNTuple() = default;
 
+   // create getters to return header and footer information
+   uint32_t GetSeekHeader() const {return fSeekHeader;}
+   uint32_t GetNBytesHeader() const {return fNBytesHeader;}
+   uint32_t GetLenHeader() const {return fLenHeader;}
+
+   uint32_t GetSeekFooter() const {return fSeekFooter;}
+   uint32_t GetNBytesFooter() const {return fNBytesFooter;}
+   uint32_t GetLenFooter() const {return fLenFooter;}
```

Comment:
Given that these functions will also be visible by other users of the class, it would be better to leverage that `ROOT::Experimental::Internal::RNTupleTester` is a `friend struct`, i.e. can access private/protected (otherwise not visible) members.

See [here](https://github.com/root-project/root/blob/master/tree/ntuple/v7/test/ntuple_minifile.cxx#L15) for an example on how to use this.